.ci/azure/linux_arm64.yml

```diff
@@ -17,13 +21,31 @@ jobs:
     system.debug: true
     VSTS_HTTP_RETRY: 5
     VSTS_HTTP_TIMEOUT: 200
+    PYTHON_ARM_VERSION: "3.8.12"
+    OPENVINO_ARCH: 'aarch64'
+    NUM_PROC: 1
     BUILD_TYPE: Release
-    REPO_DIR: $(Build.Repository.LocalPath)
-    OPENVINO_REPO_DIR: $(REPO_DIR)/../openvino
-    OPENCV_REPO_DIR: $(REPO_DIR)/../opencv
+    REPO_OPENVINO: $(REPO_CONTRIB)/../openvino
+    REPO_CONTRIB: $(Build.Repository.LocalPath)
+    REPO_OPENCV: $(REPO_CONTRIB)/../opencv
+    REPO_OMZ: $(REPO_CONTRIB)/../open_model_zoo
+    BUILD_PYTHON: $(WORK_DIR)/build_python 
+    BUILD_OPENVINO: $(WORK_DIR)/build
+    BUILD_OPENVINO_PYTHON: $(WORK_DIR)/build_openvino_python 
+    BUILD_OPENCV: $(WORK_DIR)/build_opencv
+    BUILD_OMZ: $(WORK_DIR)/build_omz
+    INSTALL_PYTHON: $(WORK_DIR)/install_python 
+    INSTALL_OPENCV: $(WORK_DIR)/install_opencv
+    INSTALL_OPENVINO: $(WORK_DIR)/install_openvino
+    INSTALL_OMZ: $(WORK_DIR)/install_omz
+    FULL_PACKAGE: $(WORK_DIR)/full_package
+    RUNTIME_OPENVINO: $(WORK_DIR)/runtime_openvino
     WORK_DIR: $(Pipeline.Workspace)/_w
-    BUILD_DIR: $(WORK_DIR)/build
-    BUILD_DIR_OPENCV: $(WORK_DIR)/build_opencv
+    SHARE_DIR: /mount/cinfsshare/onnxtestdata
+    TMP_DIR: /mnt/tmp
+    OCV_TMP_DIR: /mnt/tmp_ocv
+    OV_CCACHE_DIR: $(SHARE_DIR)/ccache/master/linux_arm64_contrib
```

Comment:
use the same as openvino. There are no diff

Reply:
done